Firmware_Bootstrap_Loader/faradaybsl/createtiscripter.py

```diff
@@ -78,7 +78,8 @@ def CreateOutputFile(self):
             final_crc = hex(self.CalcTiTxtCrc16(self.section_data_index[i]))
 
             #Script Format
-            script_index_crc = "CRC_CHECK "+"0x"+str(final_addr)+' '+str(final_len)+' '+str(final_crc)
+            script_index_crc = "CRC_CHECK 0x{} {} {}".format(
```

Comment:
OK So it was a folder permission or something on my computer. I copied all of the BSL folder contents into my master and ran it with success as well.

I made a fake firmware version with "5555" for firmware to verify firmware loading.

![image](https://cloud.githubusercontent.com/assets/339144/24741185/c8a01c64-1a59-11e7-85a2-7b55760d0636.png)